dwave_networkx/algorithms/tests/test_canonicalization.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import collections
+
+import dimod
+import dwave_networkx as dnx
+import networkx as nx
+
+from dwave_networkx.algorithms.canonicalization import rooted_tile, canonical_chimera_labeling, _chimera_shore_size
+from dwave_networkx.generators.chimera import chimera_coordinates
+
+
+class TestRootedTile(unittest.TestCase):
+
+    def test_C33_tiles(self):
+        C33 = dnx.chimera_graph(3, 3, 4)
+
+        for root in range(0, len(C33), 8):
+
+            horiz, vert = rooted_tile(C33.adj, root, 4)
+
+            self.assertEqual(horiz, set(range(root, root+4)))
+            self.assertEqual(vert, set(range(root+4, root+8)))
+
+
+class TestCanonicalChimeraLabeling(unittest.TestCase):
+    def test_tile_identity(self):
+        C1 = dnx.chimera_graph(1)
+        coord = chimera_coordinates(1, 1, 4)
+
+        labels = canonical_chimera_labeling(C1)
+        labels = {v: coord.int(labels[v]) for v in labels}
+
+        G = nx.relabel_nodes(C1, labels, copy=True)
```

Comment:
Are we following math notation for variable naming? As in capital `G` for graphs and capital `C` for chimera graphs?

Reply:
Following NetworkX's notation for this library. Also why we use a different docstring convention.